scripts/internal/replay_events_csv.go

```
@@ -242,15 +242,15 @@ func postSingleEvent(client *http.Client, endpoint, apiKey string, event *dto.In
 			time.Sleep(replayInitialBackoff * time.Duration(attempt))
 		}
 
-		req, err := http.NewRequest(http.MethodPost, endpoint, bytes.NewReader(body))
+		req, err := http.NewRequest(http.MethodPost, endpoint, bytes.NewReader(body)) // #nosec G704 -- endpoint from env, replays to own API
```

golangci-lint (2.13.2)

[error] net/http.NewRequest must not be called. use net/http.NewRequestWithContext

(noctx)